src/_includes/previously/script.js

```diff
@@ -0,0 +1,14 @@
+// Image lazy loading
+const observer = new IntersectionObserver(entries => {
+  for (const { isIntersecting, target } of entries) {
+    if (isIntersecting === false) continue;
+    target.src = target.dataset.src;
+    target.style.opacity = 1;
+    observer.unobserve(target);
+  }
+});
```

Comment:
```suggestion
}, {margin: "10%"});
```

To trigger the load slightly _before_ the image comes into view?